oss-internship-2020/libtiff/test/helper.cc

```diff
@@ -18,20 +18,24 @@
 #include "sandboxed_api/sandbox2/util/fileops.h"
 #include "sandboxed_api/sandbox2/util/path.h"
 
-std::string GetImagesFolder() {
+std::string GetFilePath(const absl::string_view filename) {
   std::string cwd = sandbox2::file_util::fileops::GetCWD();
   auto find = cwd.rfind("/build");
```

Comment:
You can use `sandboxed_api/sandbox2/util/runfiles.h`'s `sandbox2::GetDataDependencyFilePath()` instead of relying on `/build` being part of the current directory.

Note that you can specify any directory as the build directory when using CMake.